packages/babel-plugin-jest-hoist/src/index.ts

```diff
@@ -133,14 +134,16 @@ FUNCTIONS.mock = args => {
   return false;
 };
 
-FUNCTIONS.unmock = args => args.length === 1 && args[0].isStringLiteral();
-FUNCTIONS.deepUnmock = args => args.length === 1 && args[0].isStringLiteral();
+FUNCTIONS.unmock = (args: Array<any>) =>
+  args.length === 1 && args[0].isStringLiteral();
+FUNCTIONS.deepUnmock = (args: Array<any>) =>
+  args.length === 1 && args[0].isStringLiteral();
 
-FUNCTIONS.disableAutomock = FUNCTIONS.enableAutomock = args =>
+FUNCTIONS.disableAutomock = FUNCTIONS.enableAutomock = (args: any) =>
   args.length === 0;
 
-module.exports = () => {
-  const shouldHoistExpression = expr => {
+export default () => {
```

Comment:
```suggestion
export = () => {
```

That way we don't change the interface